docs/faq.md

```diff
@@ -311,4 +311,18 @@ let populated = GREYCondition(name: "Wait for UICollectionView to populate", blo
 }).waitWithTimeout(20.0)
 
 GREYAssertTrue(populated, reason: "Failed to populate UICollectionView in 20 seconds")
-```
\ No newline at end of file
+```
+
+**How do I match AX=N elements?**
```

Comment:
**How do I match AX=N elements?** =>
**How do I match elements that are denoted with "AX=N" in the view hierarchy?**